content/en/docs/ops/troubleshooting/istioctl-analyze/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Analyze your cluster and local configuration with istioctl analyze
+description: Shows you how to use istioctl analyze to identify potential issues with your configuration.
+weight: 90
+keywords: [istioctl, debugging, kubernetes]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Getting started in under a minute
+
+`Istioctl analyze` is a powerful Istio diagnostic command that you can get started with in no time, and on any cluster.
+
+First, download the latest `istioctl` into the current folder using one bash command:
```

Comment:
```suggestion
If you're using Istio 1.3 or earlier, you'll first need to download a newer version of `istioctl`:
```

Reply:
We actually would prefer users to always download very latest as analysers . I added a comment to explain that.